### PR TITLE
Privacy addition

### PR DESCRIPTION
**Pull request by mbouzi**
- link to storj privacy policy in footer
- add privacy page for possible future use

### CODE REVIEW

File: content/privacy-policy/privacyPolicy.md [hunk 1]
```diff
@@ -1,3 +1,456 @@
...
+
+<b>8.&nbsp;&nbsp;International Users</b>
+
+<u>Personal Data Transfers Outside of the EEA</u>
+
+<p>Storj may transfer some of your personal information outside of the EEA. Storj may transmit some of your personal information to a country where the data protection laws may not provide a level of protection equivalent to the laws in your jurisdiction, including the United States. As required by applicable law, Storj will provide an adequate level of protection for your personal data using various means, including, where appropriate:</p>
+
+<ul>
+	<li><p>relying on a formal decision by the European Commission that a certain country ensures an adequate level of protection for personal data (a full list of such decisions may be accessed online <a target="_blank class="link" href="http://ec.europa.eu/justice/data-protection/international-transfers/adequacy/index_en.htm">here</a>.</li></p>
+	<li><p>entering into appropriate data transfer agreements based on language approved by the European Commission, such as the Standard Contractual Clauses (2010/87/EC and/or 2004/915/EC), which are available upon request at <a href="mailto:legal@storj.io">legal@storj.io;</a></li></p>
```
Comment (boshevski) [inline content/privacy-policy/privacyPolicy.md]: Link is missing class="link".

File: content/privacy-policy/privacyPolicy.md [hunk 2]
```diff
@@ -1,3 +1,456 @@
...
+
+<b>10.&nbsp;&nbsp;Additional Information for Nevada Residents<b>
+
+<p>Under Nevada law, certain Nevada residents may opt out of the “sale” of “covered information” (as those terms are defined under Nevada law) where the sale allows the person buying it to license or sell such information to additional persons. “Covered information” includes first and last name, address, email address, phone number, social security number, or an identifier that allows a specific person to be contacted either physically or online.</p>
+
+<p>Storj does not sell consumer information as defined under Nevada law.     Notwithstanding, if you are a Nevada resident who has purchased or leased goods or services from us, Nevada law permits you to submit a request to opt out of the sale of your covered information.  To do so, you may email your request to <a href="mailto:nevadarequests@storj.io">nevadarequests@storj.io</a>. Please note we will take reasonable steps to verify your identity and the authenticity of the request.</p>
```
Comment (boshevski) [inline content/privacy-policy/privacyPolicy.md]: Link is missing class="link".

File: content/privacy-policy/privacyPolicy.md [hunk 3]
```diff
@@ -1,3 +1,456 @@
...
+
+<b>14.&nbsp;&nbsp;Children's Privacy</b>
+
+<p>The Services are not intended for children under 13 years of age and we do not knowingly collect, maintain, or use Personal Information from children under 13 years of age. We will take reasonable steps to delete personal information (as defined by the United States Children’s Online Privacy Protection Act) as soon as reasonably possible if we learn that we have inadvertently collected it from children under the age of 13 without parental consent.</p>
+
+<p>If you learn that your child has provided us with Personal Information without your consent, you may alert us at <a href="mailto:legal@storj.io">legal@storj.io</a>. If we learn that we have collected any Personal Information from children under 13, we will promptly take steps to delete such information and terminate the child’s account.</p>
```
Comment (boshevski) [inline content/privacy-policy/privacyPolicy.md]: Link is missing class="link".

File: content/privacy-policy/privacyPolicy.md [hunk 4]
```diff
@@ -1,3 +1,456 @@
...
+
+
+<b>15.&nbsp;&nbsp;Changes to our Privacy Policy</b>
+
+<p>We reserve the right to amend this Privacy Policy at any time. We will make the revised Privacy Policy accessible through the Services, so you should review the Policy periodically. You can know if the Privacy Policy has changed since the last time you reviewed it by checking the “Effective Date” at the beginning of this Privacy Policy. If we make a material change to this Privacy Policy, we will provide you with notice in accordance with legal requirements. By continuing to use the Services, you are confirming that you have read and understood the latest version of this Privacy Policy.</p>
+
+
+<b>16.&nbsp;&nbsp;Contact Information</b>
+
+<p>If you wish to contact us or have any questions about, or complaints in relation to, this Privacy Policy, please email us at <a href="mailto:legal@storj.io">legal@storj.io</a>. Please note that your communications with us may not necessarily be secure. Therefore, please do not include credit card information and/or other sensitive information in your communications with us</p>
```
Comment (boshevski) [inline content/privacy-policy/privacyPolicy.md]: Link is missing class="link".

File: layouts/partials/footer.html [hunk 5]
```diff
@@ -12,6 +12,7 @@ <h5 class="text-white">Legal</h5>
 			<ul class="list-unstyled">
 				<!-- <li><a class="footer__link" href="#">Privacy Policy</a></li> -->
 				<li><a class="footer__link" href="/terms-of-service">Terms of Service</a></li>
+				<li><a class="footer__link" href="https://storj.io/privacy-policy/">Privacy Policy</a></li>
```
Comment (boshevski) [inline layouts/partials/footer.html:15]: Link points to the storj.io privacy policy.

Comment (mbouzi) [reply layouts/partials/footer.html:15]: doing that for now, we're not sure if any terms need to be changed for the tardigrade specific one yet. right now this is just copy and paste from storj

Comment (boshevski) [reply layouts/partials/footer.html:15]: Ok, makes sense now.

**Pull request merged**